projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -333,7 +340,10 @@ export class AuthService implements OnDestroy {
         if (!isLoading) {
           this.refreshState$.next();
         }
-        const target = result?.appState?.target ?? '/';
+        const appState = result.appState;
+        const target = appState?.target ?? '/';
+
+        this.appStateSubject$.next(appState);
```

Comment:
Even though the comments on the `appState$` mention it emits (if any), yet it emits even if `appState` is `null` or `undefined`.